keps/sig-apps/3335-statefulset-slice/README.md

```diff
@@ -729,7 +735,7 @@ Longer term, we may want to require automated upgrade/rollback tests, but we
 are missing a bunch of machinery and tooling and can't do that now.
 -->
 
-Manual upgrade->downgrade->upgrade scenario (to be validated):
+A manual upgrade->downgrade->upgrade scenario was performed:
```

Comment:
Nit: in the section below, specifically `###### How can someone using this feature know that it is working for their instance?` please make sure to check the box next to `- [ ] Other (treat as last resort)` I almost missed that question :wink:

Reply:
Updated